src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpoint.java

```diff
@@ -79,6 +99,22 @@ public CaseSearchResult searchCases(
         return result;
     }
 
+    private List<String> getCaseTypeIds(List<String> caseTypeIds) {
```

Comment:
wonder whether it's a good practice leaving the parameter type as a List<String> now that `*` is a possible value. 
Wonder whether it's more correct to change it to a String and then parse the string.
Undecided. Let's discuss about it